tests/migrations/test_migrations_initial_squash/0001_initial.py

```diff
@@ -0,0 +1,13 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    replaces = [
+        ('default', '0001_initial'),
+        ('something_else', '0001_initial')
```

Comment:
Are we able to reproduce this issue without referring to non-installed and cross apps migrations?

The `Migration.replaces` attribute is an implementation detail of migration squashing (only mention in the docs is from the [_Squashing migrations_ section](https://docs.djangoproject.com/en/3.1/topics/migrations/#squashing-migrations)) and in no way `squashmigrations` will generate a migration with cross app references like it's the case here as it works on a per-app basis.

Unless you can also reproduce the above issue with the following changes I think the issue might actually be due to an abuse of the API

```suggestion
        ('migrations', '0001_initial'),
        ('migrations', '0002_replaced'),
```

Reply:
Hi @charettes ,

I have applied the changes you suggested and was still able to replicate the issue when the `applied` attribute check **isn't** present, resulting in the following stack trace:

```
ERROR: test_showmigrations_list_squashed (migrations.test_commands.MigrateTests)
Ensures a migration that doesn't have an applied time stamp can still be
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/path_to/django/django/test/utils.py", line 382, in inner
    return func(*args, **kwargs)
  File "/path_to/django/tests/migrations/test_commands.py", line 523, in test_showmigrations_list_squashed
    call_command("showmigrations", format='list', stdout=out, no_color=True, verbosity=2)
  File "/path_to/django/django/core/management/__init__.py", line 181, in call_command
    return command.execute(*args, **defaults)
  File "/path_to/django/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/path_to/django/django/core/management/commands/showmigrations.py", line 52, in handle
    return self.show_list(connection, options['app_label'])
  File "/path_to/django/django/core/management/commands/showmigrations.py", line 97, in show_list
    output += ' (applied at %s)' % applied_migration.applied.strftime('%Y-%m-%d %H:%M:%S')
AttributeError: 'Migration' object has no attribute 'applied'
```

When utilising the proposed attribute check before accessing `applied`, the tests pass as they did originally.

Please review the latest commit to ensure you are satisfied with the changes I made to reproduce the issue you have raised.

Thanks